test/xcode-specs.js

```diff
@@ -70,7 +81,7 @@ describe('xcode @skip-linux', () => {
     let path = await xcode.getAutomationTraceTemplatePath();
 
     should.exist(path);
-    fileExists(path).should.eventually.be.true;
+    fs.exists(path).should.eventually.be.true;
```

Comment:
I think it needs await (should probably add async to the method in appium-support)